src/Profile/PathValue.cpp

```diff
@@ -49,6 +49,28 @@ ProfileMap::GetPath(const char *key) const
   return ExpandLocalPath(Path(buffer));
 }
 
+std::vector<AllocatedPath>
+ProfileMap::GetMultiplePaths(const char *key) const
+{
+	std::vector<AllocatedPath> paths;
+	TCHAR buffer[MAX_PATH];
+
+	if(!Get(key, buffer, ARRAY_SIZE(buffer)))
+		return paths;
+	
+	if(StringIsEmpty(buffer))
+		return paths;
+
+	char *path = strtok(buffer, ",");
```

Comment:
This will fail to compile on Windows.
And please, never use `strtok()`.
What if the filename has a comma?

Reply:
There is an inline `StringToken()` in `StrinAPI.hxx`, but that just calls `strtok()` so I don't suppose I can use that? That also does not fix the issue when there is a comma in the filename.

Another alternative would be using `boost::tokenizer` with each file surrounded by quotes. Problem here is that `KeyValueFileReader::Read` stops reading after it finds a closing `"`. I don't really want to change that function simply to allow this niche functionality. 

Is simply disallowing commas in filenames a possibility?

Reply:
No, please don't use that either. This is just a wrapper to reduce the pain of using it in legacy code.
You could use `IterableSplitString`, which needs just a string view, doesn't modify the string and is zero-copy.

Reply:
You could disallow commas in filenames, but then your could should be robust, and have lots of additional checks. I don't see any of that in your code submission.
I wouldn't want to disallow commas. I don't think that's a good separator.